### PR TITLE
Add in copy edits

### PR DESCRIPTION
**Pull request by gedankenstuecke**
Pages/sections done: 

- nav bar
- front page
- footer
- registration page 
- share experience
- fixed capitalisation in trigger labels
- experience list page
- my stories page
- "about autism" page

### CODE REVIEW

File: server/apps/main/templates/main/about_us.html [hunk 1]
```diff
@@ -22,15 +22,17 @@
           <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
              href="#about-autspaces">About AutSPACEs<span class="badge badge-pill"></span></a>
           <a class="list-group-item list-group-item-action  d-flex justify-content-between align-items-center"
-             href="#platform-usage">How to use the Platform<span class="badge badge-pill"></span></a>
+             href="#platform-usage">How to use the AutSPACEs<span class="badge badge-pill"></span></a>
```
Comment (GeorgiaHCA) [inline server/apps/main/templates/main/about_us.html:25]: href="#platform-usage">How to use AutSPACEs<span class="badge badge-pill"></span></a>

File: server/apps/main/templates/main/about_us.html [hunk 2]
```diff
@@ -41,115 +43,203 @@
...
         </div>
         <div class="about-content-section">
-          <h5 class="heading-blue" id="data-usage"><em></em>Data Usage</h5>
-          <p>We will <b>never</b> sell your data.</p>
+          <h5 class="heading-blue" id="data-usage"><em></em>Data usage</h5>
+          <p>
+            We will never sell your data. We believe your data should belong to you, so it’s your choice what to do with it.
+          </p>
 
           <p>We believe that your data should belong to you, so it’s your choice what to do with it.</p>
```
Comment (GeorgiaHCA) [inline server/apps/main/templates/main/about_us.html:143]: Is this a repeat of the line above?

File: server/apps/main/templates/main/experiences_page.html [hunk 3]
```diff
@@ -52,12 +53,11 @@ <h1 class="big-heading">View Stories</h1>
...
           <div class="col-md-2">
-            <button class="btn btn-primary mb-2" type="submit">Search Stories</button>
+            <button class="btn btn-primary mb-2" type="submit">Search stories</button>
           </div>
         </div>
-        <div class="trigger-warning-message"> We have excluded potentially
-          triggering content, please select any categories that you would be
-          comfortable seeing
+        <div class="trigger-warning-message">
+          We have excluded potentially triggering content. Please select any categories you would be comfortable seeing:
```
Comment (GeorgiaHCA) [inline server/apps/main/templates/main/experiences_page.html:60]: Is it worth putting this in bold or making it stand out in some way given lots of people in user testing glanced over it and got it the wrong way round?

**Review by GeorgiaHCA — APPROVED** [review]
Fantastic! It reads so well. Thanks for doing the PR @gedankenstuecke ! Just a few tiny comments.

File: server/apps/main/templates/main/about_us.html (hunk 1 above)
Comment (GeorgiaHCA) [reply server/apps/main/templates/main/about_us.html:25]: (extra "the" in there)

**Comment by gedankenstuecke** [comment]
sorry, i'm not done yet, it's still WIP!

**Comment by gedankenstuecke** [comment]
okay, all done now and ready for final review! 🎉

File: server/apps/main/templates/main/experiences_page.html (hunk 3 above)
Comment (gedankenstuecke) [reply server/apps/main/templates/main/experiences_page.html]: I made it bold!

**Review by GeorgiaHCA — APPROVED** [review]
Looks perfect! Really clear and consistent. No more changes I can see to make. Thanks @gedankenstuecke !!

**Comment by gedankenstuecke** [comment]
Perfect, I'll merge it in and deploy so that we can see it in the actual context (always easier to spot things that way)

**Pull request merged**